README.md

```diff
@@ -210,11 +228,56 @@ public String getFoo() {
     // This error will not cause the span to have error tag
     throw new InvalidArgumentException();
 }
+```
+
+### Track span context across threads
+
+Use the Java Agent to track span context across thread boundaries. Provide
+explicit marker for spans to automatically propagate them when using Java's
+standard concurrency tools.  If you already have access to the current scope
+(e.g., from an ``activate()`` call), set the `async propagation` flag on the
+span like this:
+
+```java
+import com.signalfx.tracing.context.TraceScope;
+
+// ...
 
+    Span span = GlobalTracer.get().buildSpan("my-operation").start();
+    try (Scope sc = GlobalTracer.get().scopeManager().activate(span, true)) {
+        // The below cast will always work as long as you haven't set a custom tracer.
+        ((TraceScope) scope).setAsyncPropagation(true);
+        // ... Dispatch work to a Java thread.
+        // Any methods calls in the new thread will have their active scope set to the current one.
+    }
+```
+
+If you don't have access to the scope that determines whether the operation
+should be continued across threads, you can get it from the `GlobalTracer`
+like this:
+
+```java
+import com.signalfx.tracing.context.TraceScope;
+
+// ...
 
+    Span span = GlobalTracer.get().buildSpan("my-operation").start();
+    try (Scope sc = GlobalTracer.get().scopeManager().activate(span, true)) {
```

Comment:
The point of this example to show that they haven't created their own span/scope or have an immediate handle on it.  Only the inner line is intended (as I think the original example was written).

Reply:
what exactly needs to change here?

Reply:
remove lines 268-269,274